test/basic_test.cc

```diff
@@ -96,4 +96,23 @@ void BM_empty_stop_start(benchmark::State& state) {
 BENCHMARK(BM_empty_stop_start);
 BENCHMARK(BM_empty_stop_start)->ThreadPerCpu();
 
+
```

Comment:
should all the tests (but this one) switch to the range loop to show it's the default?

Reply:
Could I do that as a separate refactoring, and use it in my assignment? :-D

Reply:
ha!

yes, that's fine.